osu.Game/Skinning/RulesetSkinProvidingContainer.cs

```diff
@@ -81,7 +81,7 @@ protected override void RefreshSources()
                 }
             }
 
-            int lastDefaultSkinIndex = sources.IndexOf(sources.OfType<DefaultSkin>().LastOrDefault());
+            int lastDefaultSkinIndex = sources.IndexOf(sources.OfType<TrianglesSkin>().LastOrDefault());
```

Comment:
Would be further correct to use `SkinManager.DefaultSkin`, but that's to be decided when the new default skin is added.